deploy/newStable/0_coreBorrow.ts

```diff
@@ -0,0 +1,71 @@
+import { ChainId, registry } from '@angleprotocol/sdk';
```

Comment:
Why are those files different from the previous ones?

Reply:
There are some nuances for the deployment of a new stable and I figured out even if it's duplicate code, it'd simplify a lot the deployment process.
Like when you need to deploy a newStable you care about what's in the newStable folder and if you want to deploy for the first time on a chain, you need to look at what's in there